brave_testrun_generator.py

```diff
@@ -391,7 +395,20 @@ def android_testruns():
   Androidx86list = ['x86', 'release-notes/exclude', 'tests']
 
   if args.test is None:
-    android_repo.create_issue(title=Androidx86title,body=bigline,assignee="srirambv",milestone=android_milestone[android_key] ,labels=Androidx86list)
+    android_repo.create_issue(title=Androidx86title,body=bigline,assignee="LaurenWags",milestone=android_milestone[android_key] ,labels=Androidx86list)
+
+  print("Android Tab Checklist:")
+  bigline = "## Per release specialty tests\n"
+  for line in android_tab_checklist:
+    bigline += line + "\n"
+  bigline = bigline + android_template
+  print(bigline)
+  print("")
+  AndroidTabtitle = "Manual test run on Android Tab  for " + android_key
+  AndroidTablist = ['ARM', 'release-notes/exclude', 'tests']
```

Comment:
Looks like you're adding labels to the Tablet test run for ARM, release-notes/exclude and tests. Do we want the ARM label to be something like 'Tablet' instead? ARM is probably fine since I'm guessing the tablet is an ARM device, just wondering.

Reply:
Well I was just going by the device architecture similar to mobile but I can update it to `ARM/Tab` to be more specific. WDYT?

Reply:
Device architecture sgtm. We can always update later if it becomes a problem.